bin-src/tasks/gitInfo.ts

```diff
@@ -175,6 +209,16 @@ export const setGitInfo = async (ctx: Context, task: Task) => {
             .join('\n')}`
         );
       }
+
+      const packageManifestDependencyChanges = await Promise.all(
+        packageManifestChanges.map(async ({ commit: buildCommit, changedFiles }) => {
+          return getPackageManagerChanges(buildCommit, changedFiles);
+        })
+      );
+
+      const flattenedChanges = packageManifestDependencyChanges.flat();
+      // remove duplicate entries (if have multiple ancestors and both changed the same package.json, for example)
+      ctx.git.changedPackageManifests = Array.from(new Set(flattenedChanges));
```

Comment:
Probably a good idea to extract all of this to a separate function (and file), along with the other stuff (L183-196) above and the `getPackageManagerChanges` function. I'm fine if you then test it as a whole rather than test `getPackageManagerChanges` separately.